oak-search-elastic/src/test/java/org/apache/jackrabbit/oak/plugins/index/elasticsearch/ElasticsearchPropertyIndexTest.java

```diff
@@ -77,7 +77,8 @@ protected ContentRepository createRepository() {
         ElasticsearchConnection coordinate = new ElasticsearchConnection(
                 ElasticsearchConnection.DEFAULT_SCHEME,
                 elastic.getContainerIpAddress(),
-                elastic.getMappedPort(ElasticsearchConnection.DEFAULT_PORT)
+                elastic.getMappedPort(ElasticsearchConnection.DEFAULT_PORT),
+                "" + System.currentTimeMillis()
```

Comment:
getUniqueIndexName here as well?

Reply:
This is a different bundle so we can't use that utility method from benchmark bundle here. If needed in future we can create one utility method here as well.

Reply:
I will do this as part of the overall refactoring of elastic tests under this issue https://issues.apache.org/jira/browse/OAK-9014